ROADMAP.md

```diff
@@ -24,11 +24,13 @@
 
 ## 0.14.0
 
-- [x] Deprecate the old menu folder. #2443
-- [ ] Deprecate usage of JSON in favor of composability.
+- [x] #2443 Deprecate the old menu folder.
+- [ ] #2456 Deprecate usage of JSON in favor of composability.
 - [ ] Documentation versioning (This is a must before removing deprecations).
 - [ ] Enforce eslint rules.
+- [ ] #2433 Auto-generate Docs from source-code.
+- [ ] #2493 Use higher order components across the library to abstract themes passed down from context.
```

Comment:
Sounds good. I would also add improve the theme geneneration.
The latest issue this zIndex could be prevented with a more powerfull ThemeManager.